src/core/client/stream/common/UserBox/UserBoxContainer.tsx

```diff
@@ -122,12 +122,21 @@ export class UserBoxContainer extends Component<Props> {
       return null;
     }
 
+    const winLeft =
+      window.screenLeft !== undefined ? window.screenLeft : window.screenX;
+    const winTop =
+      window.screenTop !== undefined ? window.screenTop : window.screenY;
+
+    const popupWidth = 350;
+    const popupLeft = winLeft + window.outerWidth / 2 - popupWidth / 2;
+    const popupTop = winTop + 100;
+
     return (
       <>
         <Popup
           href={this.authUrl}
           title="Coral Auth"
-          features="menubar=0,resizable=0,width=350,height=450,top=100,left=500"
+          features={`menubar=0,resizable=0,width=350,height=450,top=${popupTop},left=${popupLeft}}`}
```

Comment:
I'd suggest instead to make `features` accept an object with `Record<string, any>` where by default, the `top` and `left` are set as you've defined above for the `Popover` component.

Reply:
Can do!